Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/MarkerDetection/MarkerDetectorCoordinateService.cs

```diff
@@ -160,7 +160,10 @@ protected override async Task OnDiscoverCoordinatesAsync(CancellationToken cance
                         await Task.Delay(TimeSpan.FromSeconds(1), cancellationToken).IgnoreCancellation();
                     }
 
-                    DebugLog("We have been told to stop");
+                    if (cancellationToken.IsCancellationRequested)
+                    {
+                        DebugLog("We have been told to stop");
+                    }
```

Comment:
Yeah, good catch.

Reply:
FWIW, I'll probably clear out a lot of this logging at some point. But for now this was kind of confusing.